doc/manpage-podboat.asciidoc

```diff
@@ -67,9 +67,13 @@ dotfiles::
 _$HOME/.newsboat/queue_
 
 XDG::
-        _$HOME/.config/newsboat/config_
+        _$XDG_CONFIG_HOME/newsboat/config_
 +
-_$HOME/.local/share/newsboat/queue_
+_$XDG_DATA_HOME/newsboat/queue_
++
+Note: if `XDG_CONFIG_HOME` environment variable is not set, Newsboat behaves as
```

Comment:
Same here: "if **the** `XDG_CONFIG_HOME`…"

And I reckon this should read "Podboat" rather than "Newsboat", shouldn't it?

Reply:
Right, changed to "Podboat", thanks!

Technically, the chapter-files.asciidoc should say "Newsboat and Podboat" everywhere, but that's hardly readable, so I opted to say just "Newsboat" everywhere.